### PR TITLE
copy PR templates

### PR DESCRIPTION
**Pull request by proth**

### CODE REVIEW

File: setup-common-project-files.sh [hunk 1]
```diff
@@ -28,13 +32,15 @@ isDebugConfiguration=false
...
-	echo "This script performs all common project setup scripts by default. You can optionally pass the projects base directory path as argument. Exceptions can be declared with the flags:" 
-	echo -e "$noSwiftlintFlag" 
-	echo -e "$noCodebeatFlag" 
-	echo -e "\nUsage:\n$ $0 $noCodebeatFlag\n" 
-	echo -e "or:\n$ $0 $noCodebeatFlag /Code/Project/Test\n" 
-} 
+function display_usage () {
+	echo "This script performs all common project setup scripts by default. You can optionally pass the projects base directory path as argument. Exceptions can be declared with the flags:"
+	echo -e "$noSwiftlintFlag"
+	echo -e "$frameworkFlag"
```
Comment (tschob) [inline setup-common-project-files.sh]: `frameworkFlag` is no exception. I think you should modify the usage string to keep the opt-out flags and add other parameters with a new text.

File: setup-common-project-files.sh [hunk 2]
```diff
@@ -10,6 +10,8 @@
 #
 
 readonly noSwiftlintFlag="--no-swiftlint"
+readonly noPRTemplateCopyFlag="--no-pr-template-copy"
+readonly frameworkFlag="--is-framework"
```
Comment (tschob) [inline setup-common-project-files.sh]: Did you check if we can get this information from the build configuration?

Comment (proth) [inline setup-common-project-files.sh]: build config value is either Release or Debug.

File: setup-common-project-files.sh (hunk 1 above)
Comment (proth) [inline setup-common-project-files.sh]: I don't know what you just said.

**Pull request merged**